docs/docs/examples/vector_stores/BagelIndexDemo.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<p>Updating from "betabageldb" to "bagelML" to rebrand our package name.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/12594/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>